shared/php/database.php

```diff
@@ -143,4 +155,42 @@ function clear_password_reset($conn, $user_id){
   $stmt = $conn->prepare($sql);
   $stmt->bindParam(1, $user_id);
   return $stmt->execute();
+}
+
+function change_username($conn, $user_id, $new_username)
+{
+  if ($conn == null || $user_id == null || $new_username == null) {
+    return null;
+  }
+  $sql = "UPDATE user SET Username=? WHERE ID=?;";
```

Comment:
Please make sure that this is the same database  table as in one.com

Reply:
in another pr